### PR TITLE
Include needed js files inside the XBlock

### PR DESCRIPTION
**Pull request by shadinaif**
## Change description

Include needed js files inside the XBlock. See [slack](https://appsembler.slack.com/archives/C0DLX9U86/p1646828038246199)

## Type of change
- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

https://github.com/appsembler/edx-sga/pull/7
https://appsembler.atlassian.net/browse/RED-2314

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: edx_sga/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.11.0'
+__version__ = '0.11.0.appsembler.1'
```
Comment (OmarIthawi) [inline edx_sga/__init__.py:5]: Let's not put too much dots, since it may break the installation.

```suggestion
__version__ = '0.11.0.appsembler1'
```

**Review by OmarIthawi — APPROVED** [review]
Thanks @shadinaif. Looks good. 

Please change the version and test on devstack, otherwise it should be good to go.

File: edx_sga/__init__.py (hunk 1 above)
Comment (shadinaif) [reply edx_sga/__init__.py]: done

**Review by OmarIthawi — APPROVED** [review]

**Comment by OmarIthawi** [comment]
@shadinaif are tests running in this pull request correctly? Would you mind running `tox` locally until #11 is fixed?

I'm marking this pull request as `draft` until then.

**Comment by shadinaif** [comment]
Tested on devstack @OmarIthawi @thraxil , let's merge and test on staging

**Review by melvinsoft — APPROVED** [review]

**Pull request merged**